packages/app/src/app/overmind/namespaces/pro/actions.ts

```diff
@@ -32,16 +33,13 @@ export const billingAmountLoaded = ({ state }: Context) => {
   state.pro.isBillingAmountLoaded = true;
 };
 
-export const updateSelectedPlan = ({ state }: Context, plan: Plan) => {
-  state.pro.selectedPlan = plan;
-};
-
+// TODO
```

Comment:
Is it missing anything here?

Reply:
Nice catch! I think I wanted to change something about changing the interval, but then found that it should work the way it does. Butttt I can still change it to not pass the `billingInterval` because it's always `Yearly` because we can't select a plan anymore. I will update this TODO, because it is a refactor rather than something functional.